README.markdown

```diff
@@ -73,6 +73,17 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 Supports only RPM-based distros: Redhat and Centos 5/6/7.
 
+###Known Issues
+
+The puppet_agent module has several known issues on Windows. This module supports Windows Server 2003 or later.
+
+* For Windows Server 2003, only x86 is supported.
+* Because the module directory has changed, modules installed manually on individual agents cannot be found after upgrading to Puppet 4.x. You should reinstall these modules with `puppet module install`.
```

Comment:
I was only made aware of it as it relates to Windows. If this is a known issue on all platforms, I'll change it to a Known Issue apart from Windows.